assets/scss/templates/_customstyles.scss

```diff
@@ -60,8 +63,35 @@
  border: 1px solid purple;
 }
 
+
+
+// ======= Footer Area ======
+
   @media (min-width: 1200px) { 
   .container {
    max-width: 1300px;
   }
-  }
\ No newline at end of file
+  }
+
+  @media (max-width: 450px){
+    nav .container{
+      padding: 0;
+      justify-content: space-around;
+    }
+
+    .navbar > .container, .navbar > .container-fluid{
+      justify-content: normal;
+    }
+    .container {
+      padding: 35px 35px 0 ;
```

Comment:
Try not to use px. Rems and Ems are better even if the math is more difficult. 

Also why are you only setting padding on the top and right?

Reply:
Try not to use px. Rems and Ems are better even if the math is more difficult.
R/  ok What unit should i use?
i using what i is been used in these particular theme

Also why are you only setting padding on the top and right?
R/ is a short hand for left and Right and 0 top Bottom